content/en/service_management/workflows/actions_catalog/saved_actions.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Store and Reuse Actions
```

Comment:
For some reason the title is appearing as an H2 instead of a Title.

Reply:
This was happening to me in preview and I thought I'd fixed it. I'll take another look

Reply:
Fixed it. The `cascade` on the index page was causing the issue.